hubble/executor/parsers/__init__.py

```diff
@@ -126,4 +140,13 @@ def get_main_parser(parser=None):
         )
     )
 
+    set_hub_list_parser(
+        sp.add_parser(
+            'list',
+            help='show the local location of Jina executors',
```

Comment:
```suggestion
            help='display a list of local Jina executors',
```

Reply:
Agree with you, I will convert `Show the local location of` to` Display a list of local`